src/hooks/useReportActionsLoading.ts

```diff
@@ -0,0 +1,128 @@
+import {useIsFocused} from '@react-navigation/native';
+import {useCallback, useMemo, useRef} from 'react';
+import type {OnyxEntry} from 'react-native-onyx';
+import {getNewerActions, getOlderActions} from '@userActions/Report';
+import CONST from '@src/CONST';
+import type {Report, ReportAction} from '@src/types/onyx';
+import {isEmptyObject} from '@src/types/utils/EmptyObject';
+import useNetwork from './useNetwork';
+
+type UseReportActionLoadingArguments = {
+    /** The id of the current report */
+    reportID: string;
+
+    /** The id of the report Action (if specific action was linked to */
+    reportActionID?: string;
+
+    /** The id of the current report */
```

Comment:
This description seems incorrect
```suggestion
    /** The id of the current report */
```

Reply:
will fix